openedx/core/djangoapps/user_api/preferences/api.py

```diff
@@ -421,8 +422,8 @@ def _create_preference_update_error(preference_key, preference_value, error):
 
 def get_country_time_zones(country_code=None):
     """
-    Returns a list of time zones commonly used in given country
-    or list of all time zones, if country code is None.
+    Returns a sorted list of time zones commonly used in given
+    country or list of all time zones, if country code is None.
```

Comment:
I would say that common_timezones is the right thing to display to the user in a dropdown box, and is probably the right thing to validate against as well, though if someone wants to make a case for validating against all_timezones, there could be a worthwhile discussion, though it would open up new UI issues.